### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by gabrielcoder247**
### My portfolio
**Milestone 1**

In this milestone I added the following:

- HTML
- CSS
- LINTERS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- HTML
+- CSS
+- LINTERS
+
+
+
+## Live Demo
+
+[Live Demo Link](https://github.com/gabrielcoder247/my-portfolio/)
+
```
Comment (carlos-ssh) [inline README.md:22]: - Kindly fix your live demo (I tried to go at the live but sends me to another way).

File: index.html [hunk 2]
```diff
@@ -5,11 +5,194 @@
...
+                    <li>Codekit</li>
+                    <li>Codepen</li>
+                    <li>GitHub</li>
+                    <li>GitLAb</li>
+                    <li>Terminal</li>
+                </ul>
+
+            </div>
+
+        </section> -->
```
Comment (carlos-ssh) [inline index.html:193]: - Kindly remove commented code.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My-Portfolio
+
+> One paragraph statement about the project.
```
Comment (carlos-ssh) [inline README.md:5]: - Kindly add general information about your project.
- How to install your repository locally.
- how to run it.
- Kindly add a License.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @gabrielcoder247, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!
Your project, it's nice. But changes are needed. After fix these issues, request another code review from your dashboard.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Keep good practices.
- Work in your requirements project.
- Work in your Figma design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @gabrielcoder247, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @gabrielcoder247 👋 , 

# STATUS APPROVED ✔️ 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**